trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+class TrimmerDecorator < Decorator
+  def correct_name
+    str = @nameable.correct_name
+    str.length > 10 ? str[0, 10] : str
```

Comment:
- Hey, the name should return a maximum of 10 characters, ie [0, 9] as in your case is returning 11 characters. please let's update it.

Reply:
@zieeco actually with all respect you didn't notice a comma there is a difference between `str[0..9]` and `str[0,9]` the  first syntax means get all chars that begin at index 0  till index  9 so it has a total of 10 chars, but `str[0,9]` means fetch the substring that begins at index 0 and has a total length = 9  that is it

Reply:
@zieeco  an example you can find in that [article](https://www.rubyguides.com/2018/01/ruby-string-methods/#How_to_Extract_a_Substring) 
![image](https://user-images.githubusercontent.com/22921170/203457184-14bdd10c-a80b-4331-94de-f5fedb2c9b35.png)